diff --git a/resources/js/echo.js b/resources/js/echo.js
index bce9d31..09454cf 100644
--- a/resources/js/echo.js
+++ b/resources/js/echo.js
@@ -2,14 +2,14 @@ import Echo from "laravel-echo";
 import { io } from "socket.io-client";
 
 // Khởi tạo trực tiếp client
-const socket = io("https://node.laravel.tk", {
+const socket = io("https://node.tungocvan.com", {
     transports: ["websocket", "polling"],
     withCredentials: false,
 });
 
 window.io = io;
 // Tạo socket client toàn cục
-window.socket = io("https://node.laravel.tk", {
+window.socket = io("https://node.tungocvan.com", {
     transports: ["websocket", "polling"],
 });
 
@@ -17,7 +17,7 @@ window.socket = io("https://node.laravel.tk", {
 window.Echo = new Echo({
     broadcaster: "socket.io",
     client: io,
-    host: "https://node.laravel.tk",
+    host: "https://node.tungocvan.com",
 });
 
 socket.on("connect", () => {
